discord/abc.py

```diff
@@ -1515,6 +1521,11 @@ async def send(
             Whether to suppress push and desktop notifications for the message.
 
             .. versionadded:: 2.4
+        poll: :class:`Poll`
+            A poll! Cannot be sent with ``content``, ``embeds`` or ``files``.
```

Comment:
overly excited, must be dull (its documentation)
```suggestion
            A poll. Cannot be sent with ``content``, ``embeds`` or ``files``.
```

Reply:
party pooper